flexget/utils/requests.py

```diff
@@ -238,6 +238,11 @@ def request(self, method, url, *args, **kwargs):
         try:
             log.debug('Fetching URL %s with args %s and kwargs %s', url, args, kwargs)
             result = super(Session, self).request(method, url, *args, **kwargs)
+            try:
+                log.trace('Contents for URL %s start on next line', url)
+                log.trace('%s', result.text)
```

Comment:
requests/models:Request defines the text property, so this will always be available.